### PR TITLE
Adding homebrew installation to documentation

### PR DESCRIPTION
**Pull request by NathanFallet**
I added a simple paragraph in the SDK page to show how to install Flutter using Homebrew, which is a preferred method for many developers using macOS.

See https://formulae.brew.sh/cask/flutter for details about the Homebrew cask.

### CODE REVIEW

**Comment by NathanFallet** [comment]
@googlebot I signed it!

File: src/docs/get-started/install/_get-sdk.md [hunk 1]
```diff
@@ -81,6 +81,17 @@ process.
 Once you have installed any missing dependencies, run the `flutter doctor`
 command again to verify that you’ve set everything up correctly.
 
+### Downloading and installing with Homebrew
+
+If you have Homebrew installed on your machine, you can install Flutter using:
```
Comment (sfshaza2) [inline src/docs/get-started/install/_get-sdk.md:86]:
```suggestion
If you have Homebrew installed on your machine,
you can install Flutter using the following command:
```

File: src/docs/get-started/install/_get-sdk.md [hunk 2]
```diff
@@ -81,6 +81,17 @@ process.
...
+
+```terminal
+$ brew install --cask flutter
+```
+
+Then, run `flutter doctor`. That will let you know if there are
+other dependencies you need to install to use Flutter (e.g. the Android SDK).
```
Comment (sfshaza2) [inline src/docs/get-started/install/_get-sdk.md:93]:
```suggestion
Then, run `flutter doctor`. That lets you know if there are
other dependencies you need to install to use Flutter, such as the Android SDK.
```

**Review by sfshaza2 — APPROVED** [review]
LGTM, with a couple of minor edits.

**Pull request merged**